exercise_solutions.tex

```diff
@@ -1441,6 +1441,245 @@ \subsection*{Solution to \cref{ex:acconn}}
+that it suffices to give a:
+\[p:\eqv{\Parens{(a,b)=(a,b)}}{\Parens{(a=a)\times (b=b)}}\]
+
+And to show that transporting $\refl{(a,b)}$ along $\ua(p)$ is equal to
+$(\refl{a}, \refl{b})$.  To construct $p$ just use \cref{thm:path-prod}. To
+show that transport respects the equality we use
+the propositional computation rule in \cref{axiom:univalence} and the definition
+of our function $p$, that is just an application of the projections and
+functoriality (see \cref{eq:path-prod}).
+
+For the inductive step we use the inductive hypothesis to get an equivalence:
+$\eqv{\Omega^n(A \times B)}{\Omega^n(A) \times \Omega^n(B)}$.
+Then we apply $\Omega(\blank)$ on both sides and use exactly the same
+propositions we used in the base
+case. This settles the inductive case, because of the inductive definition of
+$\Omega^{\suc(n)}(\blank)$.
+
+
+\subsection*{Solution to \cref{ex:contr-infinity-sphere-colim}}
+
+To solve this exercise we must first define the spheres as a type family
+$\Sn^{\blank} : \nat \to \UU$.
+Then we must define the inclusions that appear the in the diagram:
+\[ \Sn^0 \to \Sn^1 \to \Sn^2 \to\cdots \]
+
+Then we will be able to define the colimit as
+a higher inductive type.
+
+So, by induction on the natural numbers, we define $\Sn^{\blank} : \nat \to \UU$
+with the base case being the two point type $\bool$, and the inductive case
+being the iterated application of the suspension.
+
+Now we define the inclusions $i_n : \Sn^n \to \Sn^{n+1}$.
+For the base case we have to give a function $i_0 : \bool \to \susp \bool$.
+This is easy: send one point to $\north$ and the other to $\south$.
+
+For the inductive case we notice that the domain of the function $i_n$ that
+we have to define is the suspension $\susp \Sn^{n-1}$. So we can use the
+induction of $\susp \Sn^{n-1}$. The codomain is also a suspension, so we can
+use any of the constructors:
+\begin{align*}
+  \north_{n+1} &: \Sn^{n+1}\\
+  \south_{n+1} &: \Sn^{n+1}\\
+  \merid_{n+1} &:\Sn^n \to \north_{n+1} = \south_{n+1}
+\end{align*}
+to define our function.
+We send $\north_n \mapsto \north_{n+1}$ and $\south_n \mapsto \south_{n+1}$.
+Now we must give a $\susp \Sn^{n-1}$-indexed family of equalities between
+$\north_{n+1}$ and $\south_{n+1}$. By inductive hypothesis we have
+$i_{n-1} : \Sn^{n-1} \to \Sn^n$, so we can use:
+\[ \merid_{n+1} \circ i_{n-1} \]
+It is also interesting to note that this construction works in a more general
+setting: any function $f : A \to B$ induces a function $\susp f : \susp A \to \susp B$.
+
+Now is a good time to make a drawing to convince oneself that this is the
+right way to define the inclusions between consecutive spheres, and that this
+is the diagram intended in he exercise.
+
+Lets define the type $\Sn^{\infty}$ as the colimit of the diagram we just
+constructed. The constructors are:
+\begin{align*}
+  j_{\blank} &: \prd{n : \nat} \Sn^n \to \Sn^{\infty}\\
+  \glue_{\blank} &: \prd{n : \nat} \prd{x: \Sn^n} j_n(x) = j_{n+1}\circ i_n (x)
```

Comment:
I guess by `j_{n+1}\circ i_n (x)` you mean `(j_{n+1}\circ i_n)(x)`, but that's one of those things that's not immediately obvious how to parse.  I would tend to write `j_{n+1}(i_n (x))` instead.